src/systems/breadcrumbs/Breadcrumbs.cc

```diff
@@ -385,6 +400,30 @@ void Breadcrumbs::OnDeploy(const msgs::Empty &)
   IGN_PROFILE("Breadcrumbs::PreUpdate");
   {
     std::lock_guard<std::mutex> lock(this->pendingCmdsMutex);
+
+    // Check topic statistics for dropped messages
+    if (this->topicStatistics)
+    {
+      ignmsg << "Received breadcrumb deployment for " <<
+        this->modelRoot.ModelByIndex(0)->Name() << std::endl;
+      std::optional<transport::TopicStatistics> stats =
+        this->node.TopicStats(this->topic);
```

Comment:
Can we bump the required `ign-transport9` version to 9.1.0 on `CMakeLists` so a user on an older version catches the requirement early before this fails to compile?

Reply:
Done in 2f3d81042795e9fd78757bff8479d50f50ab2d80